charms/knative-serving/requirements-integration.in

```diff
@@ -2,5 +2,7 @@ aiohttp
 jinja2
 # pin juju since we use controller 2.9. Juju (app) 3.0 is end of life and py juju as well
 juju<3.0
+lightkube
```

Comment:
Good catch.  It was for the repo-level integration.txt, but I accidentally added it to both.  I've removed `serving/*integration.*` because, as you say, it is unused.